proto/gnmi/gnmi.proto

```diff
@@ -206,8 +206,8 @@ message Decimal64 {
 // ScalarArray is used to encode a mixed-type array of values.
 message ScalarArray {
   // The set of elements within the array. Each TypedValue message should
-  // specify only elements that have a field identifier of 1-7 (i.e., the
-  // values are scalar values).
+  // specify only elements that have a field identifier of 1-5 (i.e., the
+  // values are not deprecated scalar values).
```

Comment:
this seems to be (1-5,14), where 14 is a double_val

but maybe it is not necessary to call it out in the comment at all, and just say that "...only non deprecated scalar values"

Reply:
like that simplification,thanks!